src/main/java/cz/cvut/kbss/termit/service/business/VocabularyService.java

```diff
@@ -182,6 +188,16 @@ public Set<URI> getRelatedVocabularies(Vocabulary entity) {
         return repositoryService.getRelatedVocabularies(entity);
     }
 
+    @PreAuthorize("@vocabularyAuthorizationService.canRead(#entity)")
+    public List<RdfsStatement> getTermRelations(Vocabulary vocabulary) {
+        return repositoryService.getTermRelations(vocabulary);
+    }
+
+    @PreAuthorize("@vocabularyAuthorizationService.canRead(#entity)")
+    public List<RdfsStatement> getVocabularyRelations(Vocabulary vocabulary) {
+        return repositoryService.getVocabularyRelations(vocabulary, VOCABULARY_REMOVAL_IGNORED_RELATIONS);
```

Comment:
This does not seem to work. When I have a vocabulary with a single term, I can remove it. When I create a snapshot of it, I cannot remove it anymore. TermIt is telling me there is a is-snapshot-of relationship

Reply:
Was caused by incorrect subPropertyOf in old data.